### ISSUE TITLE
Local chat messages getting duplicated 

### ISSUE THREAD

**Issue by ben-roberts** [suriyun-production/mmorpg-kit-docs#204]
Local messages get 2x duplicated on all clients except for the sender. Appears to be local only, /a /p work as expected. 

1.52c kit version, running a linux server build, 2D demo project

**Comment by insthync** [comment]
Okay, I will fix it.

**Comment by insthync** [comment]
As I see, it's occuring because there is 2 colliders attached to character entity and it's still adding same entity to the list, you can fix it by add these codes to function `FindCharacters` line 1240:
```
                if (result.Contains(tempEntity))
                    continue;
```

Before `result.Add(tempEntity);`

**Comment by insthync** [comment]
This was fixed in 1.53. If this issue still occurs, you can reopen this issue post.
